clustered/server/src/main/java/org/ehcache/clustered/server/EhcachePassiveEntity.java

```diff
@@ -222,6 +224,14 @@ private void destroyServerStore(DestroyServerStore destroyServerStore) throws Cl
 
     String name = destroyServerStore.getName();
 
+    ServerStoreImpl store = ehcacheStateService.getStore(name);
+    if (store != null) {
+      TreeNode node = StatisticsManager.nodeFor(store.getPageSource());
```

Comment:
I think this code should be moved into ehcacheStateService.destroyServerStore(name);

It would be better to have all the code within the same class and classloader